src/components/Select/Select_shim.css

```
@@ -9,7 +9,11 @@
   button.neo-multiselect__header.neo-multiselect__header--no-after:after {
   display: none;
 }
-
+body .neo-multiselect__content.neo-set-keyboard-focus,
+body .neo-multiselect__content:focus-visible {
+  outline-offset: 1px;
+  outline: 1px auto #1473e6;
```

Suggfestion: Can use Neo global outline color here

```suggestion
  outline: 1px auto var(--global-outline-color);
```

---

done.